python/core/auto_generated/qgselevationmap.sip.in

```diff
@@ -31,26 +31,70 @@ in meters.
 #include "qgselevationmap.h"
 %End
   public:
+
+    enum class CombineMethod
+    {
+      HighestElevation,
+      NewerElevation,
+    };
+
+    QgsElevationMap();
+%Docstring
+Default constructor
+%End
+
     explicit QgsElevationMap( const QSize &size );
 %Docstring
 Constructs an elevation map with the given width and height
 %End
 
-    void applyEyeDomeLighting( QImage &img, int distance, float strength, float rendererScale );
+    explicit QgsElevationMap( const QImage &image );
+%Docstring
+Constructs an elevation map from an existing raw elevation ``image``
```

Comment:
Can you add some docs about what the expected format of `image ` is? I.e is it supposed to be grayscale? Or some other RGB elevation encoding?